devel/api.md

```diff
@@ -655,6 +661,58 @@ Type: `string`
 
 The interface name which current VLAN is based on.
 
+# VXLAN
+Besides basic interface properties, each VXLAN interface state also contains
+a dictionary saved in key `VXLAN.CONFIG_SUBTREE`.
+
+Example:
+
+```python
+{
+    Interface.KEY: [{
+        Interface.NAME: "eth1.101",
```

Comment:
This line is using double quotes but the other lines are using single quotes, please use a uniform style.